pkg/icingaredis/telemetry/ha.go

```diff
@@ -0,0 +1,24 @@
+package telemetry
+
+import (
+	"context"
+	"github.com/go-redis/redis/v8"
+	"github.com/icinga/icingadb/pkg/icingaredis"
+	"github.com/icinga/icingadb/pkg/logging"
+	"github.com/icinga/icingadb/pkg/utils"
+	"go.uber.org/zap"
+)
+
+var boolToIntStr = map[bool]string{false: "0", true: "1"}
+
+// ReportResponsibility updates whether and since when we're responsible (HA) in Redis for being monitored by Icinga 2.
+func ReportResponsibility(ctx context.Context, client *icingaredis.Client, logger *logging.Logger, isResponsible bool) {
+	cmd := client.XAdd(ctx, &redis.XAddArgs{
+		Stream: "icingadb:telemetry:ha",
+		MaxLen: 1,
+		Values: []string{"is-responsible", boolToIntStr[isResponsible]},
```

Comment:
How exactly do you imagine a situation and how we'd notice it this way where a state of your choice of that flag (but not limited to) would be red in the monitoring?

Reply:
I want the check to report an error when there is no known active instance. The easiest way to force this kind of situation would be shutting down the SQL server (though that should now be also be captured by the database error check). The only thing I can imagine right now would be fiddling around with clocks so that you get into the `Received heartbeat from the past` case in `ha.go`.

Reply:
> But then Icinga DB 2 would takeover. Unless... it's dead

Or unless you messed up that node in a similar way. I'd say to judge if Icinga DB is healthy in my environment, I want an answer to the question "Is some instance active?" rather than "Is this instance active?". Is there any particular issue with getting this information? I think there shouldn't as we log an "Another instance is active" message already anyways.